engine/src/edittool.cpp

```diff
@@ -38,14 +38,15 @@ along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
 #include "graphic.h"
 #include "edittool.h"
 
+#define kInvalidEditPoint MAXUINT4
```

Comment:
Because this is C++, you can safely define constants that have no runtime storage using e.g.

```c++
static const uint4 kInvalidEditPoint = MAXUINT4;
```

I recommend this instead of `#define` for situations like this.  One interesting reason is that you get better debugging and error messages because the name is resolved by the compiler rather than the preprocessor.